app/controllers/application_controller.rb

```diff
@@ -14,7 +14,7 @@ def sign_in(user)
   end
 
   def current_user
-    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    @current_user ||= User.find_by_id(session[:user_id]) if session[:user_id]
```

Comment:
What was the reasoning behind this change?  Aren't `#find` and `#find_by_id` equivalent?